### DISCUSSION TITLE
v5 Angular Query Experimental RxJS Query Results

### DISCUSSION THREAD

**Discussion by DakotaP22** [TanStack/query#6474] [Q&A] [answered]
Is there any way to access the query results as an observable? I would like to connect my server state to some complex local state using a second library which requires Subjects/Observables as inputs but I do not see any options for this. Don't get me wrong, I love Signals, and ultimately the second library I am using will expose the data as Signals to the components, but it would be great for authoring to be able to work with queries as Subjects/Observables for situations such as this

**Comment by arnoud-dv** [comment, marked as answer]
You can easily convert any signals using [toObservable](https://angular.dev/guide/signals/rxjs-interop#toobservable).

Currently exploring ways to further improve developer experience working with Angular Query and observables. But the RxJS interop from Angular is very good.

**Reply by DakotaP22** [reply]
That is a fair answer, and ultimately what I did to link these things together. I am just coming from using the @ngneat/query implementation and wasn't sure if there was an observable exposed directly or not. Everything is still working great with toObservable!
